modules/ossm-about-configuring-a-gateway-to-accept-ingress-traffic.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+
+// gateways/ossm-about-gateways.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-configuring-a-gateway-to-accept-ingress-traffic_{context}"]
+= About configuring a gateway installed using gateway injection to accept ingress traffic
+:context: ossm-about-configuring-a-gateway-to-accept-ingress-traffic
+
+When you install a gateway using gateway injection you can configure it to receive ingress traffic using the {istio} `Gateway` and `VirtualService` resources in combination. The {istio} `Gateway` resource describes a load balancer operating at the edge of the mesh that receives incoming or outgoing HTTP/TCP connections. The `Gateway` specification describes a set of ports that should be exposed, the type of protocol to use, and the Server Name Indication (SNI) configuration for the load balancer. `VirtualServices` define routing rules to apply to an {istio} `Gateway`, similar to how you can use `VirtualServices` to define routing rules for internal mesh traffic.
+
+In the following example an {istio} Gateway resource configures a gateway proxy to act as a load balancer. This configuration exposes port 443 (HTTPS) for the host, `bookinfo.com`. The configuration applies in all namespaces for gateway pods with the labeled `istio: ingressgateway`.
```

Comment:
```suggestion
In the following example an {istio} Gateway resource configures a gateway proxy to act as an entry point for external traffic. This configuration exposes port 443 (HTTPS) for the host, `bookinfo.com`. The configuration applies to pods labelled with `istio: ingressgateway` which typically correspond to ingress gateway pods. The tls mode is configured as SIMPLE, so it would terminate the incoming HTTPS traffic using the provided certificate and private key.
```

Reply:
done